### ISSUE TITLE
Media-window not as a popup

### ISSUE THREAD

**Issue by practical-solutions** [turnermm/ckgedit#526]
Trying to add an image using firefox results in the media selection opening in the same window (and not in a popup), thus making it impossible to add a media file.

![image](https://user-images.githubusercontent.com/60605145/75628541-68f36080-5bda-11ea-84ea-40c72d5413b2.png)

**Comment by turnermm** [comment]
> Trying to add an image using firefox results in the media selection opening in the same window (and not in a popup), thus making it impossible to add a media file.

Not clear.  If by this you mean copy and paste, this still works.  See https://www.dokuwiki.org/plugin:ckgedit#copy_and_paste

**Comment by practical-solutions** [comment]
What I mean, is that (using Firefox/Chrome), as you click on the "search server" button, the dialog for chosing the media file opens up within the same tab. When you chose a media file, there is no returning to the editor....

![image](https://user-images.githubusercontent.com/60605145/76251546-75fdf880-6247-11ea-8b72-4846f80b5f38.png)

![image](https://user-images.githubusercontent.com/60605145/76251667-b3fb1c80-6247-11ea-93c0-c3c96ada52d2.png)

![image](https://user-images.githubusercontent.com/60605145/76251827-03414d00-6248-11ea-8b37-078fbbc00b84.png)

**Comment by turnermm** [comment]
This is not characteristic of ckgedit, and I don't know where to begin to reproduce it.  It's never been reported before, possibly it is something unique to your configuration.  Check for javascript conflicts originating in one of your other plugins or with your template using your browser's console.

**Comment by practical-solutions** [comment]
Thank you very much for your hint. Indeed, I deactivated all of the other plugins. CKGEdit worked again as usual. So I activated all of the other plugins one by one, checking if the above error occured. The solution: **There seems to be a conflict with the [draw.io-Plugin](https://www.dokuwiki.org/plugin:drawio)**. Activating it causes the problem I described here. I added this [issue](https://github.com/lejmr/dokuwiki-plugin-drawio/issues/16) on the plugins page.
